test/babelish/test_csv2base.rb

```diff
@@ -10,38 +10,38 @@ def test_initialize
       :excluded_states => excluded_states,
       :state_column => state_column,
       :keys_column => keys_column
-      })
+    })
 
-      assert_equal csv_filename, converter.csv_filename
-      assert_equal excluded_states, converter.excluded_states
-      assert_equal state_column, converter.state_column
-      assert_equal keys_column, converter.keys_column
-    end
+    assert_equal csv_filename, converter.csv_filename
+    assert_equal excluded_states, converter.excluded_states
+    assert_equal state_column, converter.state_column
+    assert_equal keys_column, converter.keys_column
+  end
 
-    def test_initialize_with_default_values
-      converter = Babelish::Csv2Base.new("file.csv", {"English" => ["en"]})
-      assert_not_nil converter.csv_filename
-    end
-    
-    def test_initialize_with_custom_separator
-      converter = Babelish::Csv2Base.new("file.csv", {"English" => ["en"]}, {
-        :csv_separator => ";"
-        })
-      assert_not_nil converter.csv_filename
-    end
+  def test_initialize_with_default_values
+    converter = Babelish::Csv2Base.new("file.csv", {"English" => ["en"]})
+    assert_not_nil converter.csv_filename
+  end
 
-    def test_create_file_from_path
-      test_file = "test_file.txt"
-      Babelish::Csv2Base.new("file.csv", {"English" => ["en"]}).create_file_from_path test_file
-      assert File.exist?(test_file)
+  def test_initialize_with_custom_separator
+    converter = Babelish::Csv2Base.new("file.csv", {"English" => ["en"]}, {
+      :csv_separator => ";"
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.